packages/office-ui-fabric-react/src/components/Nav/Nav.types.ts

```diff
@@ -97,10 +97,15 @@ export interface INavProps {
   ariaLabel?: string;
 
   /**
-   * (Optional) The nav container aria label.
+   * (Optional) The nav container aria label. If not provided, the aria label will default to the link name.
    */
   expandButtonAriaLabel?: string;
 
+  /**
+   * (Optional) If true, the link name will be prepended to `expandButtonAriaLabel`.
+   */
+  expandButtonAriaLabelUseName?: boolean;
```

Comment:
Adding this prop because we need to include the name of the currently focused item in `expandButtonAriaLabel`, which currently allows for only a single generic label for the whole Nav. But I didn't want to forcefully include the link name and potentially interfere with existing consumers' aria-labels.

Let me know if there's a better way to do this, or a better name for the prop

Reply:
Removed new prop